InternetShop/src/main/java/dao/UserDaoImpl.java

```diff
@@ -6,18 +6,37 @@
 import java.util.Map;
 
 public class UserDaoImpl implements UserDao {
-    private Map<String, User> usersList = new HashMap<>();
+
+    private Map<String, User> users = new HashMap<>();
```

Comment:
May be not obvious, but map should be static, otherwise, you may en up with lot of "database instances" in service/menu layers